src/builder.py

```diff
@@ -131,16 +131,6 @@ def cmake(self, args=[]):
         self._builder.cmake(builddir, self._path, args)
         return FileHandle(self._builder, builddir)
 
-        # Skip directory names.
```

Comment:
Wow. That's nasty. Looks like that was left behind when we moved this code from `builder.py` to `distfile.py`.